gulpfile.js

```diff
@@ -1,98 +0,0 @@
-var gulp        = require('gulp'),
```

Comment:
What is this used for? You sure it's absolutely useless?

Reply:
Yeah, pretty sure gulpfiles aren't being used in the build process